notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p> </p><p>Γ is the symbol for a vector field. --&gt;&gt;Not exactly: Γ(TM) is the symbol for the <strong><u>space</u></strong> of vector fields. <strong><u>One element of</u></strong> Γ(TM) is a vector field, but Γ(TM) is not a vector field. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>done :D</p>